domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/Attribution.java

```diff
@@ -0,0 +1,383 @@
+                    ((isAuthority == isProducer) && (isAuthority == isOperator) &&
+                            (originalIsProducerInteger == null || originalIsProducerInteger == 0))
+            ) {
+                final String entityId = attributionId + "; " + agencyId + "; " + routeId + "; " + tripId + "; " +
+                        organizationName + "; " + isProducer + "; " + isOperator + "; " + isAuthority + "; " +
+                        attributionUrl + "; " + attributionEmail + "; " + attributionPhone;
+
+                if (organizationName == null) {
+                    noticeCollection.add(new MissingRequiredValueNotice("attributions.txt",
+                            "organization_name",
+                            entityId));
+                }
+                if (originalIsOperatorInteger != null
+                        && (originalIsOperatorInteger < 0 || originalIsOperatorInteger > 1)) {
+                    noticeCollection.add(new IntegerFieldValueOutOfRangeNotice("attributions.txt",
+                            "is_operator", entityId, 0, 1, originalIsOperatorInteger));
+                }
+                if (originalIsAuthorityInteger != null
+                        && (originalIsAuthorityInteger < 0 || originalIsAuthorityInteger > 1)) {
+                    noticeCollection.add(new IntegerFieldValueOutOfRangeNotice("attributions.txt",
+                            "is_authority", entityId, 0, 1, originalIsAuthorityInteger));
+                }
+                if (originalIsProducerInteger != null
+                        && (originalIsProducerInteger < 0 || originalIsProducerInteger > 1)) {
+                    noticeCollection.add(new IntegerFieldValueOutOfRangeNotice("attributions.txt",
+                            "is_producer", entityId, 0, 1, originalIsProducerInteger));
+                }
+                if ((isAuthority == isProducer) && (isAuthority == isOperator) &&
+                        (originalIsProducerInteger == null || originalIsProducerInteger == 0)) {
+                    noticeCollection.add(
+                            new IllegalFieldValueCombination("attributions.txt", "is_producer",
+                                    "is_authority; is_operator",
+                                    entityId));
+                }
+                return new EntityBuildResult<>(noticeCollection);
+            } else {
+                return new EntityBuildResult<>(new Attribution(attributionId, agencyId, routeId, tripId,
+                        organizationName, isProducer, isOperator, isAuthority, attributionUrl, attributionEmail,
+                        attributionPhone));
+            }
+        }
+    }
+
+    /**
+     * Returns the key used to map {@code Attribution}
+     *
+     * @return  the key used to map {@link Attribution}
+     */
+    public String getAttributionKey() {
+        return getAttributionId() + "; " + getAgencyId() + "; " + getRouteId() + "; " + getTripId() + "; "
```

Comment:
Same as FareRule, see https://github.com/MobilityData/gtfs-validator/pull/72#discussion_r438366355 and https://github.com/MobilityData/gtfs-validator/pull/72#discussion_r438369254

Reply:
As discussed, for now we use a static method to generate the key.

Reply:
See 6604871d9210650ebc6ed618d8afb109d66cb44b